### DISCUSSION TITLE
Acquiring Neuropixel probes data in Bonsai

### DISCUSSION THREAD

**Discussion by cspurandare** [bonsai-rx/bonsai#1269] [Q&A]
Hello,
We have been using OpenEphys acquisition board to acquire silicone probe data (Diagnostic Biochips) from the Bonsai GUI (using the RHD2000EvalBoard node in Bonsai), and it has worked well for us. We were hoping to now start using Neuropixel probes, connected to the NI chassis and its system. 
I tried installing the package "https://packages.nuget.org/packages/Bonsai.Neuropix/0.1.0" but I dont think its working. Is there any other way of setting up the Neuropixel acquisition through Bonsai? 
So I was wondering if there is support for acquiring Neuropixel data (from National Instruments intermediates) through Bonsai, and if yes, how can I set it up. 
Thanks in advance,
Chinmay

**Comment by bruno-f-cruz** [comment]
Heya.

Some of these questions have been answered in #1254.
What error are you getting ?
cheers,
Bruno
